apps/api/src/app/events/usecases/map-trigger-recipients/map-trigger-recipients.use-case.ts

```diff
@@ -86,11 +91,13 @@ export class MapTriggerRecipients {
     userId: UserId,
     recipients: TriggerRecipients
   ): Promise<ISubscribersDefine[]> {
-    /*
-     * TODO: We should manage the env variables from the config and not process.env
-     * https://github.com/motdotla/dotenv/issues/51
-     */
-    if (process.env.FF_IS_TOPIC_NOTIFICATION_ENABLED === 'true') {
```

Comment:
No need for spare checks in the environment process. We now can centralise everything in the feature flags module.